lib/maintainers/cli.rb

```diff
@@ -110,7 +110,7 @@ def parse(args)
             opts.on("-v", "--verbose", "verbosity") do |v|
               options[:verbose] = v
             end
-            opts.on("-d", "--details [repos|people]", "detailed report") do |v|
+            opts.on("-d", "--details [repos|people|lists]", "detailed report") do |v|
```

Comment:
We should update the help at line 56 (I was a little surprised it's not generated from opts).